package.json

```
@@ -80,5 +80,8 @@
 		"ignore": [
 			".github/workflows"
 		]
+	},
+	"dependencies": {
+		"is-buffer": "1"
```

```suggestion
		"is-buffer": "^1.1.6"
```
also let's move the deps section to right above the dev deps section